src/stdlib/base.bt

```diff
@@ -687,9 +687,9 @@ macro ppid() {
 
 // :variant string pcomm()
 // :variant string pcomm(struct task_struct * task)
-// Get the name of the parent process for the passed task or the current task if called without arguments.
+// Get the name of the parent process for the passed task or the current task if called without arguments. This is an alias for (task->group_leader->comm).
 macro pcomm(task) {
-    task->real_parent->comm
+    task->group_leader->comm
```

Comment:
`group_leader` is just the task group leader, i.e. the main thread in the current process. The documentation says "get the name of the **parent process** for the passed task or the current task"

It's inherently ambiguous: is it `parent_of(passed_task) || current_task` or `parent_of(passed_task || current_task)`

(But I don't think this satisfies either)

Perhaps the documentation is wrong? It should just be "Get the name of the process for the passed task ..."?

Reply:
Yeah, I'll just update the docs. Good call.